packages/scandipwa/src/route/MyAccount/MyAccount.container.js

```diff
@@ -486,17 +489,18 @@ export class MyAccountContainer extends PureComponent {
     }
 
     handleLocked() {
-        const { baseLinkUrl, logout } = this.props;
+        const {
+            logout, updateIsLocked, baseLinkUrl, showNotification
+        } = this.props;
 
         const path = baseLinkUrl
             ? appendWithStoreCode(ACCOUNT_LOGIN_URL)
-            : replace(/\/customer\/account\/.*/, ACCOUNT_LOGIN_URL);
+            : replace(/\/customer\/account\/?.*/i, ACCOUNT_LOGIN_URL);
 
-        history.push({
-            pathname: path,
-            state: { isFromLocked: true }
-        });
+        history.replace({ pathname: path, state: { isFromLocked: true } });
+        showNotification('error', __(LOCKED_ACCOUNT_ERROR_MESSAGE));
```

Comment:
`__(LOCKED_ACCOUNT_ERROR_MESSAGE)` this won't work; `__(...)` function requires first parameter to be direct string otherwise it will not be added to translation files.

In this example you should move translation to const.... `export const LOCKED_ACCOUNT_ERROR_MESSAGE = __("...")`
and then use `showNotification('error', LOCKED_ACCOUNT_ERROR_MESSAGE);`

Reply:
Done